lti_consumer/lti_xblock.py

```diff
@@ -846,9 +848,8 @@ def resource_link_id(self):
         i4x-2-3-lti-31de800015cf4afb973356dbe81496df this part of resource_link_id:
         makes resource_link_id to be unique among courses inside same system.
         """
-        return str(urllib.parse.quote(
-            f"{self.runtime.hostname}-{self.location.html_id()}"  # pylint: disable=no-member
-        ))
+        hostname = self.runtime.service(self, 'hostname').get_hostname()
```

Comment:
@xitij2000, did you test the whole PR? Because this should break - we have decided not to introduce this service, but rather retrieve this value directly from `settings.LMS_BASE`. I thought we have already reflected it in this PR, but it doesn't seem to be the case.

Reply:
I did not test the whole PR, just the PR with my changes only to make sure my changes did not break anything. 

I'm sorry I think I misunderstood the scope here. Should I test against this PR and update previous commits as well?

Reply:
@xitij2000, I now see that it wasn't clear from the ticket's description. Please do.